docs/index.md

```diff
@@ -24,41 +24,49 @@ A suite of utilities for AWS Lambda functions to ease adopting best practices su
 * **Lambda Layer (arm64)**: [**arn:aws:lambda:{region}:017000801446:layer:AWSLambdaPowertoolsPythonV2-Arm64:23**](#){: .copyMe}:clipboard:
-* **PyPi**: **`pip install "aws-lambda-powertools"`**
+* **Pip**: **[`pip install "aws-lambda-powertools"`](#){: .copyMe}:clipboard:**
 
-???+ info "Some utilities require additional dependencies"
-    You can stop reading if you're using Lambda Layer.
-
-    [Tracer](./core/tracer.md){target="_blank"}, [Validation](./utilities/validation.md){target="_blank"} and [Parser](./utilities/parser.md){target="_blank"} require additional dependencies. If you prefer to install all of them, use `pip install "aws-lambda-powertools[all]"`.
+??? question "Using Pip? You might need to install additional dependencies."
+    [**Tracer**](./core/tracer.md){target="_blank"}, [**Validation**](./utilities/validation.md){target="_blank"} and [**Parser**](./utilities/parser.md){target="_blank"} require additional dependencies. If you prefer to install all of them, use [**`pip install "aws-lambda-powertools[all]"`**](#){: .copyMe}:clipboard:.
 
     For example:
 
-    * [Tracer](./core/tracer.md#install){target="_blank"}: **`pip install "aws-lambda-powertools[tracer]"`**
-    * [Validation](./utilities/validation.md#install){target="_blank"}: **`pip install "aws-lambda-powertools[validation]"`**
-    * [Parser](./utilities/parser.md#install){target="_blank"}: **`pip install "aws-lambda-powertools[parser]"`**
-    * [Tracer](./core/tracer.md#install){target="_blank"} and [Parser](./utilities/parser.md#install){target="_blank"}: **`pip install "aws-lambda-powertools[tracer,parser]"`**
+    * **Tracer**: **[`pip install "aws-lambda-powertools[tracer]"`](#){: .copyMe}:clipboard:**
+    * **Validation**: **[`pip install "aws-lambda-powertools[validation]"`](#){: .copyMe}:clipboard:**
+    * **Parser**: **[`pip install "aws-lambda-powertools[parser]"`](#){: .copyMe}:clipboard:**
+    * **Tracer** and **Parser**: **[`pip install "aws-lambda-powertools[tracer,parser]"`](#){: .copyMe}:clipboard:**
 
 ### Local development
 
-Powertools relies on the AWS SDK bundled in the Lambda runtime. This helps us achieve an optimal package size and initialization.
+!!! info "Using Powertools via Lambda Layer? Simply add [**`"aws-lambda-powertools[all]"`**](#){: .copyMe}:clipboard: as a development dependency."
+
+Powertools relies on the [AWS SDK bundled in the Lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-python.html){target="_blank"}. This helps us achieve an optimal package size and initialization.
+
+However, when developing locally, you might want to have IDE autocompletion, or run your tests suite locally without any emulation like [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html){target="_blank"}. For this to work, you need to install AWS SDK as a development dependency (not as a production dependency):
```

Comment:
Can't we just simply say "However, when developing locally, you need to install AWS SDK as a development dependency (not as a production dependency)"?

I cannot think of any general scenario where developing locally without AWS SDK makes sense. Or maybe you were just enumerating scenarios where having the AWS SDK locally helps, which makes sense too.

Reply:
The latter :) We have a few customers that aren't professional developers, so we try to raise awareness of why doing X could be helpful for the uninitiated.

That said, I'm gonna try another less verbose way to do that

Reply:
Shorten the first part as you suggested, and added a long form under a banner that is folded by default to be less distracting. For anyone wanting to learn more, they can simply click to collapse to know the theory behind it.

<img width="1187" alt="image" src="https://user-images.githubusercontent.com/3340292/223159360-c99577ef-be04-4289-8a0a-2586e69fe740.png">